blog/2022-10-26-test-report/index.md

```diff
@@ -0,0 +1,468 @@
+|-----------------------------|---------|---------|---------|------|
+| cluster_schedule            | 12      | 15      | 32      | N/A  |
+| resource_distribution(Push) | 706     | 899     | 1298    | 2000 |
+| resource_distribution(Pull) | 612     | 881     | 989     | 2000 |
+
+### Push Mode
+
+Etcd latency:
+
+![etcd latency](./img/push_etcd_latency.png)
+
+Resource Usage:
+
+![resource total](./img/push_total.png)
+
+![cpu](./img/push_cpu.png)
+
+![memory](./img/push_memory.png)
+
+### Pull Mode
+
+Etcd latency:
+
+![etcd latency](./img/pull_etcd_latency.png)
+
+Resource Usage:
+
+![resource total](./img/pull_total.png)
+
+![cpu](./img/pull_cpu.png)
+
+![memory](./img/pull_memory.png)
+
+Karmada-agent in the member cluster with 5k Node and 2w Pod costs 40m CPU(cores) and 266Mi Memory(bytes).
+
+## Conclusion and Analysis
+
+From the preceding test results, the API Call Latency and Resource Distribute Latency meet the SLIs/SLOs above. And the resources consumed by the system during the whole process are in a controllable range.
+Therefore, **Karmada can stably support 100 clusters with 500000 nodes and more than two million pods**. In production, Karmada effectively supports the management of hundreds of huge clusters.
+Next, we will analyze the data of each indicator in detail.
+
+### Separation of Concerns: Resource Templates and Policies
+
+Karmada uses Kubernetes Native APIs for federated resource templates and reusable Policy APIs for cluster scheduling strategies. It not only makes it integrate with the Kubernetes ecosystem, but also greatly reduces the number of resources on the control plane.
+Based on this, the number of resources for the control plane does not depend on how many clusters on the control plane, but depends on numbers of multi-cluster applications.
+
+Karmada's arthrecture inherits the simplicity and scalability of Kubernetes. Karmada-apiserver as the entry point of the control plane is like kube-apiserver in Kubernetes. You can optimize the component in multi-cluster scenarios using the parameters you need in single cluster configuration.
+
+During the entire resource distribution process, API call latency is within a resonable range.
+
+### Cluster Registry and Resource Distribution
+
+In Karmada 1.3, we provides the ability to register Pull mode clusters based on Bootstrap tokens, which not only simplifies the cluster registration process, but also facilitates access control.
+Now whether it is Push mode or Pull mode, we both can use the karmadactl CLI to complete the cluster registration. Compared with Push mode, there is a component named karmada-agent running on the member cluster side.
+
+Cluster registry latency above contains the delay for the control plane to successfully collect the member cluster status. In the process of cluster lifecycle management, Karmada will collect the cluster version, API enablement and whether the cluster is online or healthy.
```

Comment:
```suggestion
Cluster registration latency contains the latency for the control plane to successfully collect the member cluster status. In the process of cluster lifecycle management, Karmada will collect the cluster version, API enablement and whether the cluster is online or healthy.
```

Reply:
Cluster **Registry** Latency or Cluster **Registration** Latency, please choose one and use it in all the places it shows up.